usage/installer/src/main/resources/brooklyn/entity/modaclouds/modaclouds.yaml

```diff
@@ -21,8 +22,5 @@ services:
     brooklyn.config:
       modaclouds.dda.ip: $brooklyn:component("dda").attributeWhenReady("host.address")
       modaclouds.dda.port: $brooklyn:component("dda").attributeWhenReady("modaclouds.dda.port")
-      modaclouds.kb.ip: $brooklyn:component("kb").attributeWhenReady("host.address")
-      modaclouds.kb.port: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.port")
-      modaclouds.kb.dataset.path: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.path")
```

Comment:
In case that we don't use HDB then we should also remove the deployment of it. Probably we could have 2 different blueprints `modaclouds.yaml` and `modaclouds-with-hdb.yaml`.